### PR TITLE
Add signature transaction

### PR DESCRIPTION
**Pull request by axelsheva**

### CODE REVIEW

File: core/service/transaction/signature.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { IAssetSignature, Transaction } from 'shared/model/transaction';
+import { Account } from 'shared/model/account';
+import Response from 'shared/model/response';
+import config from 'shared/util/config';
+import AccountRepo from '../../repository/account';
+
+class TransactionSignatureService implements IAssetService<IAssetSignature> {
+
+    create(trs: TransactionModel<IAssetSignature>): void {
+        return;
```
Comment (Verbalist) [inline core/service/transaction/signature.ts]: return asset

File: core/service/transaction/delegate.ts [hunk 2]
```diff
@@ -9,8 +9,9 @@ import config from 'shared/util/config';
 
 class TransactionDelegateService implements IAssetService<IAssetDelegate> {
 
-    create(trs: TransactionModel<IAssetDelegate>): void {
+    create(trs: TransactionModel<IAssetDelegate>): IAssetDelegate {
         trs.asset.username = trs.asset.username.toLowerCase().trim();
```
Comment (Verbalist) [inline core/service/transaction/delegate.ts]: ??

Comment (Verbalist) [reply core/service/transaction/delegate.ts]: remove useless line and return new obj

File: core/service/transaction/signature.ts [hunk 3]
```diff
@@ -8,8 +8,8 @@ import AccountRepo from '../../repository/account';
 
 class TransactionSignatureService implements IAssetService<IAssetSignature> {
 
-    create(trs: TransactionModel<IAssetSignature>): void {
-        return;
+    create(trs: TransactionModel<IAssetSignature>): IAssetSignature {
+        return trs.asset;
```
Comment (Verbalist) [inline core/service/transaction/signature.ts]: ?

File: core/service/transaction/delegate.ts (hunk 2 above)
Comment (axelsheva) [reply core/service/transaction/delegate.ts]: Is it the useless line?

File: core/service/transaction/signature.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        }
+        return new Response<void>({ errors });
+    }
+
+    calculateFee(trs: Transaction<IAssetSignature>, sender: Account): number {
+        return config.constants.fees.secondsignature;
+    }
+
+    applyUnconfirmed(trs: Transaction<IAssetSignature>, sender: Account): void {
+        AccountRepo.updateSecondPublicKey(trs.senderAddress, trs.asset.publicKey);
```
Comment (Verbalist) [inline core/service/transaction/signature.ts]: remove it and use local sender

**Pull request merged**